### PR TITLE
CVE dogu release

### LINKED ISSUE #143: Add CVE-Release Target for Dogus
Add a target that will trigger a release process if critical cves are lower after an dogu rebuild.

### PR DESCRIPTION
**Pull request by nhinze23**
Add release target for dogus if a simple rebuild fixes critical CVEs.

Resolves #143

### CODE REVIEW

**Review by ppxl — CHANGES_REQUESTED** [review]
nice work, i think this will really hit! :D

I think the major drawback is that the shell scripts do not receive any automated testing, f. i. with BATS. I think this is easily 
possible

**Comment by ppxl** [comment]
Do not forget: Source guards

File: build/make/release.mk [hunk 1]
```diff
@@ -8,4 +8,8 @@ dogu-release: ## Start a dogu release
 
 .PHONY: go-release
 go-release: ## Start a go tool release
-	build/make/release.sh go-tool
\ No newline at end of file
+	build/make/release.sh go-tool
+
+.PHONY: dogu-cve-release
+dogu-cve-release: ## Start a dogu release of a new build if the local build fixes critical CVEs
```
[the remark this answers is not in the document]
Comment (nhinze23) [reply build/make/release.mk:14]: With these targets everyone has to import an `.env` in the Dogu dirs. I think with the actual approach it is much simpler.

File: build/make/release_cve.sh [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+set -o nounset
+set -o pipefail
+
+function diffArrays {
+  local I=("$1")
+  local J=("$2")
+  local RESULT=()
+
+  local i
+  # shellcheck disable=SC2128
```
[the remark this answers is not in the document]
Comment (nhinze23) [reply build/make/release_cve.sh:12]: Added comment

File: build/make/release.mk (hunk 1 above)
[the remark this answers is not in the document]
Comment (ppxl) [reply build/make/release.mk:14]: Okay we leave it as-is. ENVs may be used during CI while the developer may add her creds manually

**Pull request merged**